libs/net/dal/Migrations/20240311192020_1.0.130.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using TNO.DAL;
+
+#nullable disable
+
+namespace TNO.DAL.Migrations
+{
+    /// <inheritdoc />
+    public partial class _10130 : SeedMigration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            PreUp(migrationBuilder);
+            migrationBuilder.AddColumn<int>(
+                name: "list_option",
+                table: "media_type",
+                type: "integer",
+                nullable: false,
+                defaultValue: 0);
```

Comment:
default value of 0 means the default behavior will be to display `Sources` ?

Reply:
Should this be nullable?  Should it allow for both?  I can't recall how deep this rabbit hole went.

Reply:
Like mentioned above, Source is the default right now, and keeps the compatibility with the actual functionality without needing user intervention. But is the user adds a new media or update a existing one, the default value doesn't matter anymore, since it's under his control.